eng/common/instructions/azsdk-tools/local-sdk-workflow.instructions.md

```diff
@@ -0,0 +1,38 @@
+Your goal is to help guide the user to generate SDK and build SDK locally for TypeSpec changes. This is currently supported for five languages: .NET, Java, JavaScript(TypeScript), Python, and Go.
+
+## Steps to generate SDK locally from TypeSpec
+### Step 1: Tell user the steps to generate SDK locally
+- specify the language for SDK generation
+- have the corresponding azure-sdk-for-{language} repository cloned locally
+- specify the path to the cloned azure-sdk-for-{language} repository
+- specify the path to the 'tspconfig.yaml' file or the path to 'tsp-location.yaml' file.
+- use tools provided by 'azure-sdk-mcp' MCP server to generate SDK locally
+
+### Step 2: Identify the language for SDK generation
+- Prompt the user to select the language for which they want to generate the SDK locally. Supported languages are:
+    - .NET
+    - Java
+    - JavaScript (TypeScript)
+    - Python
+    - Go
+
+### Step 3: Check for existing azure-sdk-for-{language} repository
+- Prompt the user to provide the path to their cloned azure-sdk-for-{language} repository.
+- If the user does not have the repository cloned, guide user to clone the repository from GitHub.
+
+### Step 4: Validate repository path
+- Check if the provided path is valid. If not, prompt the user to provide a valid path.
+
+### Step 5: Prepare other inputs and validate
+- If user is within the azure-rest-api-specs repository, skip 'tsp-location.yaml' input and use the path to 'tspconfig.yaml' file.
```

Comment:
but if the user is in this context, then how are the SDKs going to be generated locally? shouldn't it be the other way where if the user is on the specs repo the flow should ask the user to move to the cloned repository?